sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/models/OperationDetails.java

```diff
@@ -144,6 +145,23 @@ public Map<String, String> getTags() {
     private void setTags(Map<String, String> tags) {
         this.tags = tags;
     }
+    /**
+     * Get the API version used to create this document classifier.
+     *
+     * @return the apiVersion value.
```

Comment:
In .NET we updated docs to use the term "service version" instead of "API version". Just pointing out but not sure if that's a concern for Java.